src/tests/ckeckout.spec.js

```diff
@@ -0,0 +1,65 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { login, password } = require('../credentials/credentials.json');
+
+test.describe('Test three: proceed to Checkout and verify products', () => {
+    test.beforeEach(async ({ loginPage, inventoryPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin(login, password);
+
+        await expect(inventoryPage.headerTitle).toBeVisible();
+    });
+
+    test('Proceed to Checkout, place an Order and verify products', async ({
+        inventoryPage, shopingCartPage, checkoutFirstPage, checkoutSecondPage,
+    }) => {
+        // Perform adding random quantity of random items to the Shopping Cart
+        const arrayOfRandomIndex = await inventoryPage.arrayOfRandomItemIndex();
```

Comment:
oh please rewrite arrayOfRandomItemIndex into the plain code, Im not sure its reusable at all.

Reply:
But you reuse it! Oh, please don't do that! As far as I got that code, we create and array with length of items on the page cont and fill it with numbers 0-5. Then we use these numbers as indexes but in case number of items on the page > 5, we will get an array with the first 6 elements and thats it. Then we add it in random order but its not the random items, there are still first items to be added.